src/site/_data/contributors.js

```diff
@@ -1153,6 +1153,17 @@ const contributors = {
       name: 'Google',
     },
   },
+  bmcquade: {
+    name: {
+      given: 'Bryan',
+      family: 'McQuade',
+    },
+    twitter: 'bryanmcquade',
+    country: 'US',
+    org: {
+      name: 'Google',
+    },
+  },
```

Comment:
@robdodson plz review the `contributors.js` change

P.S. this might be an exception to the JS code owner rule where you might just give me a checklist on things to look for